components/accountPreferences/PasswordPreferences.tsx

```diff
@@ -0,0 +1,108 @@
+import { usePasswordChangeMutation } from "@/saleor/api";
+import React from "react";
+import { useForm } from "react-hook-form";
+
```

Comment:
```suggestion
interface PasswordChangeFormData {
  oldPassword: string;
  newPassword: string;
  newPasswordRepeat: string;
}
```

If we create interface for the form, we can leverage type system later